docs/source/dataframe_schemas.rst

```diff
@@ -230,6 +231,42 @@ Since ``required=True`` by default, missing columns would raise an error:
     1  pandera
 
 
+.. _column validation:
+
+Stand-alone Column Validation
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In addition to being used in the context of a ``DataFrameSchema``, ``Column``
+objects can also be used to validate columns in a dataframe on its own:
+
+.. testcode:: dataframe_schemas
+
+    import pandas as pd
+    import pandera as pa
+
+    from pandera import Column, Check
+
+    df = pd.DataFrame({
+        "column1": [1, 2, 3],
+        "column2": ["a", "b", "c"],
+    })
+
+    column1 = Column(pa.Int, name="column1")
+    column2 = Column(pa.String, name="column2")
```

Comment:
can we be a little more explicit, e.g. calling this `column1_schema`, `column2_schema`?